### PR TITLE
Fix some typos and unify concepts in docs

### PR DESCRIPTION
**Pull request by carrodher**
Fix some typos and unify concepts in general regarding the documentation.
The most relevant change could be the unification of the examples by directly pointing to the docs site instead of having this documentation repeated which is more difficult to maintain

### CODE REVIEW

File: examples/text-overlay/README.md [hunk 1]
```diff
@@ -1,53 +1,3 @@
-# Hello Pipeless!
+# Multi-pose estimation example
```
Comment (miguelaeh) [inline examples/text-overlay/README.md:1]: This seems like a copy paste error

**Review by miguelaeh — COMMENTED** [review]
Thank you very much for the PR!

Could you please check the comment I left?

File: examples/text-overlay/README.md (hunk 1 above)
Comment (carrodher) [reply examples/text-overlay/README.md]: You're right, I copied it from the pose example. Now I used the same title as in the doc site https://www.pipeless.ai/docs/docs/v0/examples/text-overlay

**Review by miguelaeh — APPROVED** [review]
Awesome!

Thank you very much for the PR!

**Pull request merged**